apps/src/p5lab/constants.js

```diff
@@ -28,3 +28,8 @@ export const EMPTY_IMAGE =
   'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==';
 
 export const PlayBehavior = utils.makeEnum('ALWAYS_PLAY', 'NEVER_PLAY');
+
+export const EFFECTS_PALETTES = {
+  ocean: ['#7FD0F5', '#3FABE3', '#2C7DBB', '#1D57A0', '#144188', '#061F4B'],
+  twinkling: ['#FFC702', '#FC9103', '#F17302', '#B83604', '#7E1301'],
+};
```

Comment:
Copied from https://github.com/code-dot-org/code-dot-org/blob/mike/effects-for-spritelab/apps/src/p5lab/poetry/constants.js#L7-L28

We only needed these two palettes. The others are exclusively used in Poetry so it felt right to leave them there. I renamed to `EFFECTS_PALETTES` here since this is the only context they are used for here.

Reply:
I feel slightly less strongly about this one than the one below (where code needs to be maintained in two places instead of constants), but I propose the following in `apps/src/p5lab/poetry/constants.js` if we think it's worth having the code represent the hierarchical relationship between these two labs. 

```
import { EFFECTS_PALETTES } from '../constants';

export const PALETTES = {
  // Directly use shared palettes if needed
  ...EFFECTS_PALETTES,
  grayscale: [
    '#000000', '#333333', '#666666', '#999999', '#CCCCCC', '#EEEEEE', '#FFFFFF',
  ],
  ...
};
```